treeherder/push_health/builds.py

```diff
@@ -1,11 +1,13 @@
-from treeherder.model.models import Job
+from treeherder.model.models import Job, JobType
 from treeherder.push_health.utils import mark_failed_in_parent, get_job_results
 
 
 def get_build_failures(push, parent_push=None):
+    build_types = JobType.objects.filter(name__contains='build')
     build_results = Job.objects.filter(
         push=push,
         tier__lte=2,
+        job_type__in=build_types,
```

Comment:
@camd I made a few changes in here based on what I saw you do in your pr #6708 since some of the build results seemed off when Joel was testing this out (because its grabbing all results and then determining status in `get_job_results` so I think there were none build tests being caught in the initial commit). Could use a sanity check on this since you're more familiar with this code :)

Reply:
Not sure how much slower it might be, but we could do `name__icontains='build'` rather than just `__contains` to be sure we catch any that might be called `Build`.

I was relying on the failure of `busted` as being ONLY used for Builds.  But that may not be a valid assumption.  So I think your filter here is better.

Reply:
fwiw, looking at the `job_type` table, looks like the `builds` are all lowercase as far as I saw.  Though my search was no exhaustive.

Reply:
> Not sure how much slower it might be, but we could do `name__icontains='build'` rather than just `__contains` to be sure we catch any that might be called `Build`.
> 
> I was relying on the failure of `busted` as being ONLY used for Builds. But that may not be a valid assumption. So I think your filter here is better.

Sure, I'll try the `icontains`.